app/WebParsing/ReqParser.hs

```diff
@@ -405,12 +411,30 @@ fcesParser = do
     _ <- Parsec.spaces
     _ <- fceSeparator
     _ <- Parsec.optional $ Parsec.try includingSeparator <|> Parsec.try fromSeparator
-    req <- Parsec.try andParser <|> Parsec.try orParser
-    return $ FCES fces req
+
+    FCES fces <$> fcesModifiersParser
```

Comment:
```hs
modifiers <- fcesModifiersParser
return $ FCES fces modifiers
```
might be easier to read but the haskell linter was telling me to write it this way

Reply:
yes that's fine